kfdefs/overlays/moc/smaug/opf-superset/kfdef.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: kfdef.apps.kubeflow.org/v1
+kind: KfDef
+metadata:
+  annotations:
+    kfctl.kubeflow.io/force-delete: "false"
+  name: opendatahub
+spec:
+  applications:
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: odh-common
+      name: odh-common
+    - kustomizeConfig:
+        parameters:
```

Comment:
same as before, update these parameters to match the ones removed from override: 

```
      parameters:
      - name: superset_memory_requests
        value: 6Gi
      - name: superset_memory_limits
        value: 8Gi
      - name: superset_secret
        value: superset-custom
      - name: superset_db_secret
        value: superset-db-custom
```